mdot_rest/test/image.py

```diff
@@ -24,11 +24,11 @@ def test_convert_to_png(self):
         """ Tests that a non-PNG image is converted to PNG upon save.
         """
         # save a JPEG as the image
-        fhandle = open("{0}/files/campusmap.jpg".format(TESTROOT))
         dest = "{0}uploads/test.jpg".format(settings.MEDIA_ROOT)
-        self.uwr1.image = SimpleUploadedFile(dest, fhandle.read())
-        self.uwr1.save()
-        fhandle.close()
+
+        with open("{0}/files/campusmap.jpg".format(TESTROOT), 'rb') as fhandle:
+            self.uwr1.image = SimpleUploadedFile(dest, fhandle.read())
+            self.uwr1.save()
```

Comment:
Do a little black box testing around this with different Python versions and image types.